frontend/public/components/utils/tile-view-page.jsx

```diff
@@ -21,9 +21,18 @@ import {
 } from '@patternfly/react-core';
 
 import { history } from './router';
+import { Dropdown } from '../utils';
 
-const CATEGORY_URL_PARAM = 'category';
-const KEYWORD_URL_PARAM = 'keyword';
+export const FilterType = {
```

Comment:
Use enum here.

Reply:
Better name it `FilterTypes`.

Reply:
used const instead of enum because it's giving an error saying `'enum declarations' can only be used in a .ts file` cc @christianvogt

Reply:
oh haha... jsx... didn't pick up on that one :)
Time for an upgrade!